core/src/main/java/greencity/controller/ManagementEmployeeController.java

```diff
@@ -285,9 +283,8 @@ public ResponseEntity<Object> getAllAuthorities(@RequestParam String email) {
     })
     @PreAuthorize("@preAuthorizer.hasAuthority('EDIT_EMPLOYEES_AUTHORITIES', authentication)")
     @PutMapping("/edit-authorities")
-    public ResponseEntity<Object> editAuthorities(@Valid @RequestBody UserEmployeeAuthorityDto dto,
-        @ApiIgnore Principal principal) {
-        ubsClientService.updateEmployeesAuthorities(dto, principal.getName());
+    public ResponseEntity<Object> editAuthorities(@Valid @RequestBody UserEmployeeAuthorityDto dto) {
```

Comment:
please uopdate ApiResponses
This edpoint does not return  `response = UserEmployeeAuthorityDto.class`

Reply:
I also see that we could have `404` response code

Reply:
Done